server/userClasses/injury.py

```diff
@@ -3,11 +3,16 @@
 
 class Injury:
     NOTIFICATION_TEXT = "This is a reminder to update your injur's pain index."
-    def __init__(self): #TODO 
+    def __init__(self): 
         self.__area = "" #area of the injury (i.e. shoulder, knee)
         self.__pain = 0   #user's pain rating 1-10
         self.__injuryDate = datetime.datetime(0, 0, 0) #the date the injury occured
     
+    def __init__(self, area, pain, injuryDate):
```

Comment:
Do you have two inits?

Reply:
yeah, I thought it would be good to have two constructors.

Reply:
Ahhh, makes sense, I understand, thank you for explaining LGTM! :)